x-pack/plugins/alerts/README.md

```diff
@@ -50,6 +51,17 @@ A Kibana alert detects a condition and executes one or more actions when that co
 2. Configure feature level privileges using RBAC 
 3. Create an alert using the RESTful API [Documentation](https://www.elastic.co/guide/en/kibana/master/alerts-api-update.html) (see alerts -> create).
 
+## Alerts API keys
+
+When we create an alerts we generate a new API key.
+
+When we change, delete or manage alert availabilty we need to invalidate the old API key and create a new one (except the delete process).
```

Comment:
If we're not doing it for delete, should we leave it out of the list? `When we update, enable or disable an alert, we need to invalidate the old API key and create a new one`

Reply:
```suggestion
When we update, enable, or disable an alert, we must invalidate the old API key and create a new one.
```

Reply:
I will try try to rephrase it, because we are doing invalidation for DELETE, but not generate a new api key.